intelmq/bots/parsers/abusech/parser_ransomware.py

```diff
@@ -40,7 +40,15 @@ def process(self):
                         ev.add('classification.type', 'c&c')
                         ev.add('time.source', nrow[0] + ' UTC', overwrite=True)
                         ev.add('status', nrow[5])
-                        ev.add('source.ip', nrow[7])
+                        # in the feed sometimes a ',' is seen before srcip
+                        # perhaps it is an error on feeds part
+                        # however we have to deal with it
+                        src_ip = nrow[7]
+                        if src_ip and len(src_ip) > 8 and src_ip[0] == ',':
```

Comment:
So if the ip address is too short, you want to ignore them? That's not what this code does. IPv6 adresses can be shorter.

I reported these issues (comma, cut ip addresses) upstream at abuse.ch and those have been fixed already!

So the only thing we need to take care of now are the 0.0.0.0 IPs.